internal/project/project.go

```
@@ -0,0 +1,302 @@
+			// repository it was said to be in. Refusing beats walking on.
+			return Settings{}, nil
+		}
+		cur = parent
+	}
+}
+
+// resolve follows symlinks where it can, leaving the path alone when it
+// cannot: a directory that has since been removed is the caller's problem to
+// report, not a reason for discovery to fail here.
+func resolve(path string) string {
+	if real, err := filepath.EvalSymlinks(path); err == nil {
```

golangci-lint (2.12.2)

[error] variable real has same name as predeclared identifier

(predeclared)